api-2.0/src/main/java/org/openmrs/module/serialization/xstream/converter/CollectionCompatibility2_0.java~

```diff
@@ -36,8 +36,8 @@
 import com.thoughtworks.xstream.converters.MarshallingContext;
 import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
 
-@OpenmrsProfile(openmrsPlatformVersion = "2.0")
-public class CollectionCompatibility2_0 implements CollectionCompatibility {
+@OpenmrsProfile(openmrsPlatformVersion = "2.*")
+public class CollectionCompatibility2_x implements CollectionCompatibility {
```

Comment:
@rkorytkowski I overlooked that I've added temporary file too, thanks for notice, fixed